demo/index.html

```diff
@@ -4,54 +4,44 @@
 <head>
   <meta charset="utf-8">
   <title>highlight.js demo</title>
+  <link rel="shortcut icon" type="image/png" href="https://highlightjs.org/static/favicon.png">
```

Comment:
Long-term should we considering pulling this into our repo itself vs linking to the website? Not sure i have a strong opinion.

Reply:
I don't think it's necessary to pull it into the repo as it's only relevant once the demo is part of the website. We could use a relative path (`../favicon.png`), but that's not really better (and doesn't show up locally).

Reply:
Then we'll leave it as is for now.